auth-web/src/views/auth/HomeView.vue

```diff
@@ -17,7 +17,7 @@
       <header class="hero-banner">
```

Comment:
You know why it's called "hero banner" right?

https://www.google.com/search?q=tour+of+heroes

Reply:
😆 Angular reference?? 

I thought the reference to a hero ...something was generic!